docs/src/intro-csharp.md

```diff
@@ -189,7 +189,6 @@ See our doc on [Test Runners](./test-runners.md) to learn more about running tes
 - [Write tests using web first assertions, page fixtures and locators](./writing-tests.md)
 - [Run single tests, multiple tests, headed mode](./running-tests.md)
 - [Learn more about the NUnit and MSTest base classes](./test-runners.md)
-- [Debug tests with the Playwright Debugger](./debug.md)
```

Comment:
sounds like we can bring these links back?

Reply:
no, its getting started and it doesn't make sense  to take them off the get started path so I could link to the running tests page that has the debug section but doesnt make sense to add the link to this page